app/src/main/java/org/oppia/android/app/activity/BUILD.bazel

```
@@ -43,6 +43,32 @@ kt_android_library(
     ],
 )
 
+kt_android_library(
+    name = "injectable_auto_localized_app_compat_activity",
+    srcs = [
+        "InjectableAutoLocalizedAppCompatActivity.kt",
+    ],
+    visibility = [
+        "//app:app_visibility",
+    ],
+    deps = [
+        "injectable_app_compat_activity",
+    ],
+)
+
+kt_android_library(
+    name = "injectable_system_localized_app_compat_activity",
+    srcs = [
+        "InjectableSystemLocalizedAppCompatActivity.kt",
+    ],
+    visibility = [
+        "//app:app_visibility",
+    ],
+    deps = [
+        "injectable_app_compat_activity",
+    ],
+)
```

```suggestion
kt_android_library(
    name = "injectable_auto_localized_app_compat_activity",
    srcs = ["InjectableAutoLocalizedAppCompatActivity.kt"],
    visibility = ["//app:app_visibility"],
    deps = [":injectable_app_compat_activity"],
)

kt_android_library(
    name = "injectable_system_localized_app_compat_activity",
    srcs = ["InjectableSystemLocalizedAppCompatActivity.kt"],
    visibility = ["//app:app_visibility"],
    deps = [":injectable_app_compat_activity"],
)
```

Reduce for simplicity, but also note the ':' in front of the ``:injectable_app_compat_activity`` deps for consistency.

---

Updated.